lib/LedgerSMB/Scripts/setup.pm

```diff
@@ -417,8 +417,8 @@ sub run_backup {
         die $request->{_locale}->text('Error creating backup file');
 
     if ($request->{backup_type} eq 'email') {
-        # suppress warning of single usage of $LedgerSMB::Sysconfig::...
-        no warnings 'once';
```

Comment:
@einhverfr,  if you refer to line 423, I am saying the line will not warn.  The solution is to just remove the
unnecessary `no warnings 'once';`